src/granite_orm/transactions.cr

```diff
@@ -7,9 +7,36 @@ module Granite::ORM::Transactions
     @updated_at : Time?
     @created_at : Time?
 
+    # The import class method will run a batch INSERT statement for each model in the array
+    # the array must contain only one model class
+    # invalid model records will be skipped
+    def self.import(model_array : Array(self))
+      begin
+        @@adapter.import(table_name, primary_name, fields.dup, model_array)
+      rescue err
+        raise DB::Error.new(err.message)
+      end
+    end
+
+    def self.import(model_array : Array(self), update_on_duplicate : Bool, columns : Array(String))
```

Comment:
since you would never use `update_on_duplicate` and `ignore_on_duplicate` what if we use an enum that has options for how to handle duplicates?  maybe `duplicates: ignore` and `duplicates: update`?

Reply:
Would there be a nice way to also require columns key when using the update option?